src/modules/AnalysisExecution/components/ViewEditInsight/Toolbar/container.js

```diff
@@ -33,24 +33,28 @@ function mapStateToProps(state) {
   const studyId = get(insightData, 'analysis.study.id');
   const studyTitle = get(insightData, 'analysis.study.title');
   const analysisUrl = paths.analyses(analysisId);
+  const breadcrumbList = [
```

Comment:
Looks very similar to the block of code in `ViewEditAnalysis/Toolbar/container.js`. May be we can move it to some util?